company/vision.md

```diff
@@ -0,0 +1,80 @@
+
+- Drive the growth and success of the [Node-RED](https://nodered.org) project
+- Build a platform that becomes the standard way of running Node-RED at any scale - Be DevOps for Node-RED
+- Drive adoption of Node-RED in manufacturing
+- Build an inclusive, high-performance team
+
+#### Drive the growth and success of the Node-RED project
+
+Node-RED is at the core of what we do. Node-RED enables tens of thousands to create and maintain workflows and logic of both physical devices and digital services. Ensuring their effectiveness grows, while new users are introduced to Node-RED is vital to the success of FlowForge.
+
+FlowForge will drive the Node-RED project through:
+
+- **Deliver features** to the core of Node-RED that are stable and provide value to the community
+- **Actively participate** in the community to establish a positive reputation and level of engagement
+- **Be model citizens** for the open governance of the project. Our activities that impact the core are discussed and developed in the open.
+- **Establish close relationships** with other stakeholders of the project - those who have their own commercial interest in the success of the project. Encourage participation and engagement in the core project.
+- **Driving adoption** of Node-RED through our advocacy and our actions.
+
+#### Become the standard way of running any Node-RED at any scale - Be DevOps for Node-RED
+
+There's a wide range of deployment scenarios and FlowForge aims to build a
+platform to help each of them to be successful. FlowForge will solve challenges
+unsolved by Node-RED and push Low-Code beyond the capabilities of Node-RED.
+
+Firstly, team collaboration. Allowing Flows to be developed by multiple
+individuals, ensuring quality and correctness. Push the boundaries of Low-Code
+to allow more people to collaborate.
+
+Secondly, Node-RED is deployed in many different environments. FlowForge will allow
+everyone to achieve the best results whatever the execution context.
+
+We believe these two points will make FlowForge synonymous with Node-RED, and
+make it attractive to individual users as well as Enterprises. Value offered
+will increase exponentially as the platform scales to teams of developers and
+larger deployment scenarios.
+
+### Drive adoption of Node-RED in manufacturing
+
+Node-RED itself is already being used today in many industries to connect OT and IT environments, due to the challenge of heterogeneous protocols and demands. This shift is part of a larger trend: teams of all types are realizing that breaking down silos has compounding effects on productivity. Especially in all kinds of manufacturing and industrial companies, the way of production is changing rapidly and therefore the need to further integrate IT and OT. 
+
```

Comment:
What are we going to do to increase adoption in manufacturing. For instance are we going to provide better support for OPC-UA or Sparkplug, are are going to improve dashboard technology and make it part of FF?  We can't just assume adoption is going to come.

Reply:
See therefore the product strategy. I would not include specific steps in the vision.